src/app/(payload)/admin/affiliate/components/AffiliateSettingsTab.tsx

```
@@ -0,0 +1,311 @@
+        ...data,
+        event: parseInt(data.event),
+        affiliateUser: parseInt(data.affiliateUser),
+        promotions: data.promotions?.map((promo: any) => ({
+          ...promo,
+          promotion: parseInt(promo.promotion)
+        })) || []
+      }
+
+      console.log('formattedData', formattedData)
+
+      debugger
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)